app/config_test.go

```diff
@@ -51,20 +51,20 @@ func TestConfigListener(t *testing.T) {
 }
 
 func TestAsymmetricSigningKey(t *testing.T) {
-	th := Setup(t).InitBasic()
+	th := UnitSetup(t)
```

Comment:
It's not clear to me why this should be a unit test -- we're relying on `ensureAsymmetricSigningKey` in the course of app startup to generate this key correctly, which in turn uses the system store to read/write. I'm guessing the mock store does that now... but how do we know that that the real store still does that correctly?

Reply:
The reliability of the store is checked a different layers, one is the store layer, other is the app layer and other is the api layer. We can revert this to use integration tests here.